.github/workflows/py_checks.yml

```
@@ -106,8 +106,7 @@ jobs:
       
       # Run pyright to assert no regressions in type hinting
       - name: Run pyright on Python API
-        run: python -m pyright ./src/openvino -p ./pyproject.toml
-        working-directory: src/bindings/python
+        run: python -m pyright --verbose
```

There's now tons of unnecessary output in the `pyright` job, is there a specific reason `--verbose` is needed? Otherwise I'd opt for removing it.
```suggestion
        run: python -m pyright
```

---

why? it is better to log the test execution